packages/MarkdownEditor-Core.package/MarkdownDelimiterEmphasis.class/class/punctuationChars.st

```diff
@@ -0,0 +1,5 @@
+constants
+punctuationChars
+
+	"ASCII punctuation characters according to GFM spec. :)"
+	^ #($! $" $# $$ $% $& $' $( $) $* $+ $, $- $. $/ $: $; $< $= $> $? $@ $[ $] $\ $^ $_ $` ${ $} $| $~)
```

Comment:
This is just a reminder to ask next meeting: `Unicode` has `categoryOf:` which would make this obsolete (and would have the power of unicode). Sadly, it returns the category, like `Pf`, is encoded as a magic number.

Reply:
We decided to keep it like this and add a link to the spec